tests/imtcp-tls-ossl-basic-verifydepth.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# added 2018-04-27 by alorbach
+# This file is part of the rsyslog project, released under ASL 2.0
+. ${srcdir:=.}/diag.sh init
+export NUMMESSAGES=1000
+generate_conf
+add_conf '
+global(	defaultNetstreamDriverCAFile="'$srcdir/tls-certs/ca.pem'"
+	defaultNetstreamDriverCertFile="'$srcdir/tls-certs/cert.pem'"
+	defaultNetstreamDriverKeyFile="'$srcdir/tls-certs/key.pem'"
+)
+module(	load="../plugins/imtcp/.libs/imtcp"
+	StreamDriver.Name="ossl"
+	StreamDriver.TlsVerifyDepth="5" 
+	StreamDriver.Mode="1"
+	StreamDriver.AuthMode="x509/certvalid" )
+input(	type="imtcp"
```

Comment:
Please update the test plumbing - this is the old, racy version.  I have just fixed similar tests here: https://github.com/rsyslog/rsyslog/pull/4102/files

Pls also have all input parameters on one line - this simplifies future changes via `sed`. If there are very many parameters, at least the `type` and `port` should be in one line and in this order.

Reply:
done